tests/test_c_generator.py

```diff
@@ -246,6 +250,13 @@ def test_issue246(self):
             int array[3] = {[0] = 0, [1] = 1, [1+1] = 2};
             ''')
 
+    def test_noreturn(self):
+        self._assert_ctoc_correct(r'''
+            _Noreturn int x(void) {
```

Comment:
I'd like to see a standalone test for `_Noreturn` in `test_c_parser`. Just a small code snippet, and a test that verifies that the proper specifier is added to the function declaration.